index.bs

```diff
@@ -85,38 +90,29 @@ existing APIs, or new APIs.
 Terminology {#terminology}
 --------------------------
 
-We borrow terminology from the [[PRESENTATION-API|Presentation API]] and
-[[REMOTE-PLAYBACK|Remote Playback API]] for terms used in this document.  These
-terms are summarized here.
-
-We call the browser that is used to discover and initiate presentation of Web
-content on another device the [=controlling user agent=].  We call the
-user agent on the device rendering the Web content the
-[=receiving user agent=], or *receiver* for short.  We use
-the term [=presentation display=] to refer to the entire platform and
-responsible for implementing the *receiver*, including browser, OS, networking,
-audio and graphics.
-
-For the [[PRESENTATION-API|Presentation API]], presentation of Web content is
-initiated at the request of a [=controlling browsing context=] (or
-*controller*), which creates a [=receiving browsing context=] (or
-*presentation*) to load a [=presentation request URL=] and exchange messages
-with the resulting document.
+We use the term "agent" to mean any implementation of this protocol
+(browser, device, or otherwise), acting as a controller or a receiver.
 
-Before this can happen, the [=controlling user agent=] must determine which
-[=receivers=], if any, are compatible with the [=presentation request URL=].  This
-happens by determining the [=presentation display availability=] for the
-presentation request URL.
+We borrow terminology from the [[PRESENTATION-API|Presentation API]]. We call
+the agent that is used to discover and initiate presentation of Web content on
+another device the [=controller=] (or [=controlling user agent=] when it is a
+browser).  We call the agent on the device rendering the Web content the
+[=receiver=] or [=presentation display=] (or [=receiving user agent=] when it is
```

Comment:
A receiver could potentially launch a non-user agent.  Maybe it has specific native code to use for certain URLs that aren't user agents.

If you want to change the term Remote Playback "receiver" into something other "receiver", could we debate what that term should be and change it in a follow-up PR?  It might take a while to pick a different noun.

Reply:
I'll make a github issue to remember to discuss this (and also the word "controller")